### PR TITLE
Add option to go to checkout from PDP

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Add a button on PDP to go straight to checkout (bypass cart).

> The design is pending

## Justify why they are needed

UX improvement.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1998** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1998" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1998?utm_source=stack-comment).

File: apps/store/public/locales/en/purchase-form.json [hunk 1]
```diff
@@ -88,6 +88,7 @@
   "PRESENT_OFFER_EDIT_BUTTON": "Edit your info",
   "PRICE_CALCULATOR_SECTION_EDIT_BUTTON": "Edit",
   "PRICE_MATCH_BUBBLE_SUCCESS_TITLE": "You save {{amount}}",
+  "QUICK_CHECKOUT_BUTTON_LABEL": "Check out",
```
Comment (robinandeer) [inline apps/store/public/locales/en/purchase-form.json:91]: This label is pending

File: apps/store/src/components/ProductPage/PurchaseForm/OfferPresenter.tsx [hunk 2]
```diff
@@ -185,7 +185,14 @@ export const OfferPresenter = (props: Props) => {
               />
 
               <input type="hidden" name={FormElement.ProductOfferId} value={selectedOffer.id} />
-              <SubmitButton loading={loading} />
+
+              <Button type="submit" variant="primary" value={PageLink.checkout()} loading={loading}>
```
Comment (robinandeer) [inline apps/store/src/components/ProductPage/PurchaseForm/OfferPresenter.tsx:189]: Use submitter value to indicate that we want to go straight to checkout page

Comment (alebedev) [inline apps/store/src/components/ProductPage/PurchaseForm/OfferPresenter.tsx:189]: I remember we wanted to changes on checkout page only when you get there directly from price calculator.  Maybe the easiest way to do this is to add some query param here

**Review by alebedev — APPROVED** [review]
Very nice!

File: apps/store/src/components/ProductPage/PurchaseForm/OfferPresenter.tsx (hunk 2 above)
Comment (robinandeer) [reply apps/store/src/components/ProductPage/PurchaseForm/OfferPresenter.tsx:189]: yeah probably, will cover that in a separate PR

**Comment by robinandeer** [comment]
@robinandeer queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1998).

**Comment by robinandeer** [comment]
@robinandeer merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1998).

**Pull request merged**